CHANGELOG.md

```diff
@@ -1,37 +1,68 @@
+
+### Removed
+
+- Moved multiple utility functions to `/internal`
+
+### Fixed
+
+- Better drift detection [#106](https://github.com/UpCloudLtd/terraform-provider-upcloud/issues/106)
+- Fixed issue where a change in server storages would replace the server network interfaces and recreate the server
+- Addressed issue where a change in server networking would replace the previous storages (the template will still be created anew)
+
+## 1.0.0
+
+## 0.1.0
 
 ### Added
- - Changelog to highlight key alterations across future releases 
- - Website directory for future provider documentation
- - Vendor directory through go modules to cover CI builds
- - datasource_upcloud_hosts to view hosts data
- - datasource_upcloud_ip_addresses to retrieve account wide ip address data
- - datasource_upcloud_networks to retrieve account wide networks data
- - datasource_upcloud_tags to retrieve account wide tag data
- - datasource_upcloud_zone to retrieve specific zone details
- - datasource_upcloud_zones to retrieve account wide zone data
- - resource_upcloud_firewall_rules add to allow rules to be applied to server
- - resource_upcloud_floating_ip_address to allow the management of floating ip addresses
- - resource_upcloud_network to allow the management of networks
- - resource_upcloud_router to allow the management of routers
+
+- Changelog to highlight key alterations across future releases
+- Website directory for future provider documentation
+- Vendor directory through go modules to cover CI builds
+- datasource_upcloud_hosts to view hosts data
+- datasource_upcloud_ip_addresses to retrieve account wide ip address data
+- datasource_upcloud_networks to retrieve account wide networks data
+- datasource_upcloud_tags to retrieve account wide tag data
+- datasource_upcloud_zone to retrieve specific zone details
+- datasource_upcloud_zones to retrieve account wide zone data
+- resource_upcloud_firewall_rules add to allow rules to be applied to server
+- resource_upcloud_floating_ip_address to allow the management of floating ip addresses
+- resource_upcloud_network to allow the management of networks
+- resource_upcloud_router to allow the management of routers
 
 ### Changed
- - README and examples/README to cover local builds, setup and test execution
- - Go verison to 1.14 and against Go master branch in Travis CI
- - Travis CI file execute to execute website-test covering provider documentation
- - Provider uses Terraform Plugin SDK V2
- - resource_upcloud_server expanded with new functionality from UpCloud API 1.3
- - resource_upcloud_storage expaned with new functionalty from UpCloud API 1.3
- - resource_upclopud_tag expanded to implement read function
- 
- 
+
+- README and examples/README to cover local builds, setup and test execution
+- Go verison to 1.14 and against Go master branch in Travis CI
+- Travis CI file execute to execute website-test covering provider documentation
```

Comment:
"execute to execute", hm?

Reply:
do not like poking around old changelog entries, but guess  I ll fix the typos 😁